### ISSUE TITLE
Provide built-in help buttons to show syntax cheatsheets for searchbar's SearchEntry and quick-add task creation entry

### ISSUE THREAD

**Issue by nekohayo** [getting-things-gnome/gtg#1369]
## SearchEntry

If you test with Orca, you will notice some clarity problems regarding how those widgets are meant to be used. The search entry here is just announced as "Entry. Search here.":

<img width="1069" height="121" alt="Image" src="https://github.com/user-attachments/assets/f5a5e2d0-e326-47de-ae68-36cfcd72adc9" />

The placeholder text should instead say something like "Type here to filter the tasks list", and maybe ". Press Esc to deactivate filter." if we want to be explicit. When the user types and a search is completed, it should announce the results (see #1368).

However, GTG has advanced operators syntax available. It's documented in the user manual, but not many people will read it, and Yelp doesn't even work nowadays (you'll get a blank window), so it would be better if we added a "Syntax tips" icon button in the SearchEntry that would open a popover cheatsheet that explains/reminds the user of the syntax they can use.

## Quick add entry

<img width="830" height="123" alt="Image" src="https://github.com/user-attachments/assets/aebe3b23-b04b-4294-a298-b3808955d88a" />

Similarly to the SearchEntry, it could benefit from a "Syntax tips" button to open a popover showing the quick tasks creation's natural lanugage parsing syntax.
